.travis.yml

```diff
@@ -19,6 +19,8 @@ env:
  - PLATFORM="native_static"
  - PLATFORM="win32_dyn"
  - PLATFORM="win32_static"
+ - PLATFORM="raspberrypi_dyn"
+ - PLATFORM="raspberrypi_static"
```

Comment:
Would not be a better idea to talk about arm instead of raspberrypi? We have many other devices using arm with linux.